CHANGELOG.md

```diff
@@ -0,0 +1,15 @@
+# Changelog
+
+<!-- Instructions
+
+This changelog follows the patterns described here: <https://keepachangelog.com/en/1.0.0/>.
+
+Subheadings to categorize changes are `added, changed, deprecated, removed, fixed, security`.
+
+-->
+
+## Unreleased
+
+## 0.1 (2024-03-11)
```

Comment:
Let's have the full version number.

Also don't forget to update the date to whatever the actual release date is.

```suggestion
## 0.1.0 (2024-03-26)
```